scripts/ops/fix_arm64.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+TARGET="$(poetry env info -p)/lib/python*/site-packages/"
+
+pip install --upgrade atomics --platform=universal2 --no-deps --target $TARGET
```

Comment:
Hmmm, I feel like this should be handled by poetry.
In other words, you shouldn't need to upgrade atomics this way, but you would perhaps just need to specify `--platform` to poetry when doing the `install`. Doesn't something like that work?

Reply:
`poetry add` at least has this:
```
--platform=PLATFORM    Platforms for which the dependency must be installed.
```
so at least it looks like it's possibe to have multiple platforms supported.

Reply:
I couldn't find a way how to achieve that in poetry, but while doing more research, figured out a simpler command:

```
arch -arm64 pip install atomics --no-binary :all:
```

Would it be OK to just add it as a note to https://docs.beamerbridge.com/contributing.html?

Or perhaps add an install script, which would execute `poetry install` inside and conditionally execute above pip command?

Reply:
> I guess having install script wrapping up `yarn install` and `poetry install` and `ape compile` and fixing arm64 issue would be a nice shortcut.

Do `yarn` and `ape` need that kind of wrapper, though?
From what I understand, you only had issues with the `atomics` package -- everything else worked?

Personally, I'm leaning more toward a Mac-specific note in the docs, but it would also be nice to have feedback from people actually using Arm Macs. @fredo

Reply:
I meant following script (pseudo code):

```bash
yarn install
poetry install
if arm64 {
   arch -arm64 pip install atomics --no-binary :all:
}
poetry run ape compile
```

Then to shorten the docs with running `make install` instead of 3 separate commands.

Reply:
I think here are two different features: 

One is to improve the handling of the missing arm64 build wheel in atomics, and second creating a make install command which also handles yarn and compilation of contracts.

In general I'm in favor of shorter commands to get started. However, I see that the main benefit here would be the fix for atomics.
@panpilkarz Is there a particular reason why you also added the compile command into the install target?

Reply:
No real reason. Happy to get rid of it.